bpf-gpl/parsing.h

```diff
@@ -73,10 +88,18 @@ static CALI_BPF_INLINE int parse_packet_ip(struct cali_tc_ctx *ctx) {
 
 allow_no_fib:
 	fwd_fib_set(&ctx->fwd, false);
-	ctx->fwd.res = TC_ACT_UNSPEC;
+	if (CALI_F_XDP) {
+		ctx->fwd.res = XDP_PASS;
```

Comment:
I think fwd.res is needed for the TC cases, because the code will call forward_or_drop, which will immediately check ctx->fwd.res.  For XDP case it depends what happens after parse_packet_ip returns.

Reply:
Might be nicer to use a common set of `res` values here, and translate to XDP values higher up.

Reply:
Absolutely, using a common set of `res` makes the code nicer and allows us to re-use more code from TC program. The only challenge is that the current code heavily uses TC return codes like "TC_ACT_SHOT". If you agree, I can try to make the return code (at least) in parsing part generic and translate it later on in the callee functions. Does it make sense?